src/style.css

```diff
@@ -0,0 +1,130 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  --dark-color: #060b35;
+  --bright-color: #616070;
+  --cyan-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.product {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 10th item. Create separate file for EACH Bem-block